orttraining/orttraining/python/ort_trainer.py

```diff
@@ -299,7 +299,7 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device):
     other_export_options = {}
     # This option was added after 1.4 release.
     if LooseVersion(torch.__version__) > LooseVersion('1.4.0'):
-        other_export_options['enable_onnx_checker'] = 'False'
+        other_export_options['enable_onnx_checker'] = False
```

Comment:
Good catch. 
As string 'False' is treated as True in if statement. Maybe change to True could align with existing behavior.

Reply:
while i think bool False should be the expected value, right? @liqunfu

Reply:
Thanks for the fix. Yea, False is the intended value.